javascript-conditionals/main.js

```diff
@@ -0,0 +1,115 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('isUnderFive value:', isUnderFive(4));
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('isEven value:', isEven(4));
+
+function startsWithJ(string) {
+  if (
+    string.startsWith('J') &&
+    string.charAt(0) === string.charAt(0).toUpperCase()
+  ) {
+    return true;
+  }
+}
```

Comment:
We should also be `return`ing `false` if it doesn't start with a 'J'

Reply:
![Screenshot 2023-07-13 at 11 38 31 AM](https://github.com/julieluangpraxay/c0723-code-solutions/assets/130415142/0be1c5f7-6997-418c-be3d-4b02130def76)